docs/en/setup/service-agent/java-agent/Plugin-list.md

```diff
@@ -174,3 +174,4 @@
 - websphere-liberty-23.x
 - spring-cloud-gateway-4.x
 - spring-webflux-6.x
+- spring-webflux-6.x-webclient
```

Comment:
I think both for webclient 5.x and 6.x should be updated in https://skywalking.apache.org/docs/skywalking-java/next/en/setup/service-agent/java-agent/supported-list/

Reply:
add `  * [Spring Webflux WebClient](https://github.com/spring-projects/spring-framework/tree/main/spring-webflux) 5.x -> 6.x` in supported-list